src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/CCJStaffNotificationService.java

```diff
@@ -48,6 +48,19 @@ public void notifyStaffCCJRequestSubmitted(Claim claim) {
         emailService.sendEmail(staffEmailProperties.getSender(), prepareEmailData(claim));
     }
 
+    public EmailAttachment generateCountyCourtJudgmentPdf(Claim claim) {
```

Comment:
@geek-soft This public member of class accessed inside and out of class. I want to preempt this discussion - I think it is fine if a private method accesses public member method of a class.

Reply:
I do not like the approach because `CCJStaffNotificationService` which was meant to take care of notifications only is now exposing method that returns `EmailAttachment`. That changes the purpose of the service class significantly :(

Reply:
It also introduces unnecessary coupling between `RequestForJudgementNotificationService` and `CJStaffNotificationService` which is wrong. 

I think between solution would be duplicating the piece of code that creates email attachment. If we want to make filename the same across the app then we might extract logic to some sort of filename helper.

Reply:
Otherwise we may find ourselves redoing exact copy of codebase to produce exactly same pdf  for staff and robotics. Class can be renamed to CCJNotificationService serving staff and robotics.

Reply:
I'm not in favour of duplicating code I'm afraid. Filenames has been taken out into helper class though. We can discuss it later on when we're producing more JSONs' for RPA. In this case, I think we should move on!

Reply:
@singh-surjeet your change pollutes existing API and introduces unnecessarily coupling between unrelated components. Simple does not always mean good. I thought that you would understand necessity of making good design decisions as a "tech lead"  :(

I am happy to discuss tomorrow.

Reply:
I was assessing this case based on amount of work needed vs code reusability - design decisions cannot be generalised. Happy to discuss tomorrow and in the mean time, kindly refrain from sarcasm on public space. Not ruling out duplicating code in this case.

Reply:
Please follow the same approach as here: `uk.gov.hmcts.cmc.claimstore.rpa.DefenceResponseNotificationService#createResponsePdfAttachment`